sktime/annotation/tests/test_eagglo.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+"""Tests for E-Agglo (agglomerative clustering algorithm)."""
+
+__author__ = ["KatieBuc"]
+
+import numpy as np
+import pandas as pd
+
+from sktime.annotation.eagglo import EAGGLO
+
+
+def test_fit_default_params_univariate():
+    """Test univariate data and defualt parameters.
+
+    These numbers are generated from the original implemention
+    in R, with the following code:
+
+    set.seed(1234)
+    X <- c(rnorm(15, mean = -6), 0, rnorm(16, mean = 6))
+    X <- as.matrix(X[c(1,2,17,18)])
+    ret = e.agglo(X)
+    """
+    X = pd.DataFrame([-7.207066, -5.722571, 5.889715, 5.488990])
```

Comment:
Tests seem reasonable.  Feels a bit roundabout to expect agreement via the R implementation, but I think that the expected cluster assignments seem reasonable for the synthetic data so in the end they are good tests.